scripts/ci/automation_full_test.py

```diff
@@ -99,6 +107,97 @@
         }
 
 
+def run_git_command(cmd):
```

Comment:
Is it better to rename the function to "run_command" as the function not only support running "git command"?

Reply:
Replaced with `run_command`